### PR TITLE
Minor text changes

### PR DESCRIPTION
**Pull request by osscca**

### CODE REVIEW

File: website/ninja_web/templates/login.html [hunk 1]
```diff
@@ -33,21 +33,21 @@
             <li id="projects" class="last-of-row">
                 <h3>Project Managment</h3>
                 <img src="/media/img/feature-projects.png" alt="Feature Projects" />
-                <p>NINJA-IDE allows to manage Python Project automatically, saving descriptive information about them and letting the user to perform file managment related task in the IDE itself.
+                <p>INJA-IDE allows to manage Python projects automatically, saving descriptive information about them and letting the user perform file managment related task in the IDE itself.
```
Comment (pmourelle) [inline website/ninja_web/templates/login.html]: Missing 'N'

**Comment by pmourelle** [comment]
It's a great contribution.
Can you fix that missing N so I accept the PR?
Thanks a lot!

**Comment by osscca** [comment]
Oops... Fixed it!

**Comment by pmourelle** [comment]
Thanks osscca!

**Pull request merged**